### PR TITLE
fix(docs): updated links for cypress-tests guide to work in GitHub

### PR DESCRIPTION
**Pull request by vjvibhanshu**
Checklist:

- [Y] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).

- [Y] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [Y] My pull request targets the `main` branch of freeCodeCamp.
- [Y] I have tested these changes either locally on my machine, or GitPod.

links corrected for cypress-tests guide

### CODE REVIEW

**Review by raisedadead — APPROVED** [review]
Hi @vjvibhanshu 

Thanks for contributing to freeCodeCamp for the first time. Happy to have you here.

Some quick notes:

1. We understand you are making these changes for making the links work within GitHub, but the guidelines are meant to be read on their dedicated website: `https://contribute.freecodecamp.org` - they are way nicer there.
2. For the checklists in your OP, here is a quick guide about the syntax: https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#task-lists - must-read (30 seconds).

**Maintainers**, 

This does not break the site so I am fine with merging it.

**Review by Sboonny — APPROVED** [review]

File: docs/how-to-add-cypress-tests.md [hunk 1]
```diff
@@ -13,13 +13,13 @@ To learn how to write Cypress tests, or 'specs', please see Cypress' official [d
 ## How to run tests
 
 > [!NOTE]
-> If using GitPod, please see [Cypress-GitPod Setup](/how-to-add-cypress-tests#cypress-gitpod-setup)
+> If using GitPod, please see [Cypress-GitPod Setup](how-to-add-cypress-tests#cypress-gitpod-setup)
```
[the remark this answers is not in the document]
Comment (raisedadead) [reply docs/how-to-add-cypress-tests.md:16]: I explained the "why" in my review comments.

[the remark this answers is not in the document]
Comment (Sboonny) [reply docs/how-to-add-cypress-tests.md:16]: Nope GitHub is acting smartly and saving my comments from the previous preview

**Comment by raisedadead** [comment]
#### :sparkles: :sparkles: **AWESOME!** :sparkles: :sparkles: 

LGTM.

Thanks for this pull request, @vjvibhanshu, and for contributing to the code-base for the first time.

We will be looking forward to more contributions in the future.

Cheers and happy contributing!

**Pull request merged**